app/Http/Controllers/Legacy/Two/PostsController.php

```diff
@@ -3,13 +3,14 @@
 namespace Rogue\Http\Controllers\Legacy\Two;
 
 use Rogue\Managers\Legacy\Two\PostManager;
-use Rogue\Http\Controllers\Traits\PostRequests;
+use Rogue\Http\Requests\Legacy\Two\PostRequest;
+use Rogue\Http\Controllers\Traits\FiltersRequests;
 use Rogue\Repositories\Legacy\Two\SignupRepository;
 use Rogue\Http\Transformers\Legacy\Two\PostTransformer;
 
 class PostsController extends ApiController
 {
-    use PostRequests;
+    use FiltersRequests;
```

Comment:
just curious why you got rid of PostRequests and moved this here! Just because it is easier to read and follow?

Reply:
Yup. This trait was only used in one place now (this controller), so didn't make sense to have extracted. I also prefer to be able to see what methods a controller supports by skimming the actual controller, and not have to jump into traits to see where something might be tucked away.